src/main/java/com/uber/cadence/samples/hello/HelloSearchAttributes.java

```diff
@@ -109,8 +152,10 @@ public static void main(String[] args) {
         workflowClient.newWorkflowStub(
             HelloSearchAttributes.GreetingWorkflow.class, workflowOptions);
     // Execute a workflow waiting for it to complete.
-    String greeting = workflow.getGreeting("SearchAttributes");
+    String greeting = workflow.getGreeting("World");
```

Comment:
I would change workflow to not get any parameters and do not return anything. And compose Hello world from the values of search attributes.